optimum/exporters/openvino/convert.py

```diff
@@ -689,3 +700,34 @@ def export_tokenizer(
 
     for model, file_name in zip(converted, (OV_TOKENIZER_NAME, OV_DETOKENIZER_NAME)):
         save_model(model, output / file_name.format(suffix))
+
+
+def _add_version_info_to_model(model: Model, library_name: Optional[str] = None):
+    """
+    Add dependency versions to OpenVINO model
+    """
+    try:
+        model.set_rt_info(_transformers_version, ["optimum", "transformers_version"])
+        model.set_rt_info(_torch_version, ["optimum", "pytorch_version"])
+        model.set_rt_info(_optimum_intel_version, ["optimum", "optimum_intel_version"])
+        model.set_rt_info(_optimum_version, ["optimum", "optimum_version"])
+
+        if any("token_embeddings" in output.get_names() for output in model.outputs):
+            import sentence_transformers
+
+            model.set_rt_info(sentence_transformers.__version__, ["optimum", "sentence_transformers_version"])
+        if library_name == "diffusers":
```

Comment:
why not :
```suggestion
       if library_name == "sentence_transformers":
            import sentence_transformers

            model.set_rt_info(sentence_transformers.__version__, ["optimum", "sentence_transformers_version"])
        elif library_name == "diffusers":
```

Reply:
That was my first attempt, but that didn't work, after `optimum-cli export openvino -m BAAI/bge-base-en-v1.5 --library sentence_transformers bge-base-en-ov-st` library_name is detected as transformers by `TasksManager.infer_library_from_model`. If I copy the [config_sentence_transformers.json](https://huggingface.co/BAAI/bge-base-en-v1.5/blob/main/config_sentence_transformers.json) to the model directory, the model is detected as sentence-transformers. Should we save that file for models exported with sentence_transformers library?

Reply:
I see, an option could be to re-use the `library_name` inferred from the original model, this solution also works for me